src/components/pages/Notes/NoteGroupNav.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { connect } from "react-redux";
+import { values } from "lodash";
+import { NavLink, Route } from 'react-router-dom';
+
+import Button from "../../library/Button";
+
+
+function NoteGroupNav({ children, noteGroups, deleteNoteGroup }) {
+  return (
+    <div className="group-nav">
+      <div className="columns">
+        <div className="column is-2">
+          <aside className="menu">
+            <ul className="menu-list">
+              {noteGroups.map(noteGroup => {
+                return(
+                    <li>
+                      <NavLink to={`/notes/noteGroups/${noteGroup.id}`}>
+                        {noteGroup.name}
+                        <button
```

Comment:
Use your Button component here, and if some options are missing, add them ! You can use `classNames` to help with the option combinations